setup.py

```diff
@@ -36,6 +40,11 @@
     packages=['sanic'],
     platforms='any',
     install_requires=install_requires,
+    extras_require={
+        "all": install_requires + ujson + uvloop,
+        "ujson": install_requires + ujson,
+        "uvloop": install_requires + uvloop,
```

Comment:
For reference: https://github.com/kennethreitz/requests/blob/master/setup.py#L97-L100